### ISSUE TITLE
Newbie question on map refreshing

### ISSUE THREAD

**Issue by cbaker11** [librenms-plugins/Weathermap#34]
I'm new to weathermap in Librenms so please be gentle.  I have build a map and it is working pretty well, however, the links I created between network devices often show no traffic even though hovering over them shows significant traffic flow.  I am assuming that I have a configure item that needs to be updated by I haven't been able to find it.  Can anyone point me in the right direction?

**Comment by cbaker11** [comment]
Further investigation has lead me to think I am dealing with an rrd cache issue.  I found that if I reviewed the associated device information in the WebUI that the link info on the map would also be refreshed.  I am now keep all the ports graph page open and all my links on my map are stay refreshed.  I'll keep digging but any pointers appreciated!

**Comment by GitStoph** [comment]
@cbaker11 did you ever get this resolved? I've been having the same issue. (other than keeping the links open)

**Comment by LoveSkylark** [comment]
> Further investigation has lead me to think I am dealing with an rrd cache issue. I found that if I reviewed the associated device information in the WebUI that the link info on the map would also be refreshed. I am now keep all the ports graph page open and all my links on my map are stay refreshed. I'll keep digging but any pointers appreciated!

Yes it is a rrd cace issue, I'm working on a fix you can test it out at LoveSkylark/LibreNMS-Weathermap
